themes/ccc/components/FeatureDiscover.tsx

```diff
@@ -1,34 +1,35 @@
 import { ExternalLink } from "@/components/ExternalLink";
 import { LinkWithQuery } from "@/components/LinkWithQuery";
 import { getButtonClasses } from "@/components/atoms/button/Button";
+import { Columns } from "@/components/atoms/columns/Columns";
+import { SiteWidth } from "@/components/panels/SiteWidth";
 
 import { Feature } from "./Feature";
 
 export const FeatureDiscover = () => (
-  <Feature heading="Central knowledge-base for climate policy data" image="app_search_results.jpg" imageAlt="Screenshot of the search results">
-    <p className="text-xl my-4">
-      Discover climate laws and policies from over 196 countries, territories, and the European Union. This includes{" "}
-      <LinkWithQuery href="/framework-laws" className="underline text-blue-200 hover:text-gray-100">
-        climate framework laws
-      </LinkWithQuery>{" "}
-      and targets for every country.
-    </p>
-    <div className="flex flex-col gap-4 mt-6 lg:flex-row">
-      <LinkWithQuery
-        href="/methodology"
-        className={getButtonClasses({ color: "brand", rounded: true, className: "!bg-blue-500 hover:!bg-blue-700 !text-base" })}
-      >
-        Learn more about our database
-      </LinkWithQuery>
-      <ExternalLink
-        url="https://form.jotform.com/233131638610347"
-        className={getButtonClasses({
-          variant: "ghost",
-          className: "hover:!bg-transparent !text-base !text-blue-300 hover:!text-white underline",
-        })}
-      >
-        Download our data
-      </ExternalLink>
-    </div>
-  </Feature>
+  <SiteWidth extraClasses="pt-12 pb-4">
+    <Columns containerClasses="px-0" gridClasses="grid-cols-1 lg:grid-cols-2 gap-8">
+      <Feature heading="About us" subheading="Learn about this project" inline={true}>
+        <p className="text-lg text-gray-700 leading-relaxed mb-6">
+          The Sabin Center for Climate Change Law's Climate Litigation Database is the most comprehensive resource tracking climate change litigation
+          worldwide. It contains more than 3,000 cases that address climate change law, policy, and science.
+        </p>
+        <LinkWithQuery href="/about" className="text-blue-900 hover:text-blue-800 underline text-lg font-medium">
+          Read more →
+        </LinkWithQuery>
+      </Feature>
+      <Feature heading="Get the newsletter" subheading="Stay up to date" inline={true}>
+        <p className="text-lg text-gray-700 leading-relaxed mb-6">
+          Subscribe to the Sabin Center Climate Litigation Newsletter for twice-monthly updates. Each issue includes the latest case updates, event
+          announcements, and publication highlights.
+        </p>
+        <ExternalLink
```

Comment:
The links in the design look to be not underlined and to be bold. Also rather than the blue colour set could we use the `text-brand-darker` colour?

Reply:
Also text size 16px, which is base so we shouldn't need to specify size unless we have on a parent container